### ISSUE TITLE
cannot open link

### ISSUE THREAD

**Issue by monkeydri** [dbackowski/BackRSS#2]
When clicking one link in the available news (after I have subscribed to a RSS feeds) I see this error pop-up :

![screen shot 2017-03-03 at 14 34 23](https://cloud.githubusercontent.com/assets/18724661/23552995/e1a150f2-001e-11e7-8375-4f02081f222b.png)

```
A JavaScript error occurred in the main process

Uncaught Exception:
TypeError: Cannot read property 'openExternal' of undefined
    at WebContents.<anonymous> (/Applications/BackRSS.app/Contents/Resources/app/electron.js:182:10)
    at emitMany (events.js:127:13)
    at WebContents.emit (events.js:201:7)
    at EventEmitter.<anonymous> (/Applications/BackRSS.app/Contents/Resources/electron.asar/browser/guest-window-manager.js:145:16)
    at emitMany (events.js:127:13)
    at EventEmitter.emit (events.js:201:7)
    at WebContents.webContents.on (/Applications/BackRSS.app/Contents/Resources/electron.asar/browser/api/browser-window.js:27:13)
    at emitMany (events.js:127:13)
    at WebContents.emit (events.js:201:7)
```

The link does not open in web browser.

---

BlackRSS : 1.0.6
operating system : macOS sierra 10.12.3

**Comment by dbackowski** [comment]
Fixed in version 1.0.7

**Comment by monkeydri** [comment]
Waouh that's the quickest fix and release ever ! Thanks a lot it works great ! Although it's a bit disappointing that all subscribed feeds are deleted when the app is updated (but this is another subject).
